### PR TITLE
Discuss command syntax in tutorial's conflict resolution section

### PR DESCRIPTION
**Pull request by DamienCassou**

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -103,19 +103,38 @@ More Configuration
...
+array starting with ``"command"`` and containing paths and arguments
+to a command. For example::
+
+    [pair my_contacts]
+    ...
+    conflict_resolution = ["command", "vimdiff"]
+
+In this example, ``vimdiff <a> <b>`` will be called with ``<a>`` and
+``<b>`` two temporary files containing the conflicting files. The
+files need to be exactly the same when the command returns.
```
Comment (hobarrera) [inline docs/tutorial.rst]: Maybe it's worth specifying here that additional arguments to the command should be additional entries in the array (that's the current behaviour, right, @untitaker?)

File: docs/tutorial.rst [hunk 2]
```diff
@@ -103,19 +103,38 @@ More Configuration
...
+Option 3 requires specifying as value of ``"conflict_resolution"`` an
+array starting with ``"command"`` and containing paths and arguments
+to a command. For example::
+
+    [pair my_contacts]
+    ...
+    conflict_resolution = ["command", "vimdiff"]
+
+In this example, ``vimdiff <a> <b>`` will be called with ``<a>`` and
+``<b>`` two temporary files containing the conflicting files. The
```
Comment (untitaker) [inline docs/tutorial.rst]: I think this line is missing the word "being"

File: docs/tutorial.rst (hunk 1 above)
Comment (untitaker) [inline docs/tutorial.rst]: Yes it is, and I agree

File: docs/tutorial.rst [hunk 3]
```diff
@@ -103,19 +103,38 @@ More Configuration
...
+files need to be exactly the same when the command returns.
+
+See :ref:`_pair_config` for the reference documentation.
```
Comment (untitaker) [inline docs/tutorial.rst]: The leading underscore is not part of the label name. This link will go nowhere.

**Comment by DamienCassou** [comment]
Thanks for the quick feedback. I fixed the PR.

**Comment by untitaker** [comment]
Thanks!

**Pull request merged**